src/axolotl/core/trainers/base.py

```
@@ -20,7 +20,7 @@
     SequentialSampler,
 )
 from transformers import Trainer
-from transformers.trainer_utils import PREFIX_CHECKPOINT_DIR, seed_worker
+from transformers.trainer_utils import PREFIX_CHECKPOINT_DIR, has_length, seed_worker
```

Ruff (0.11.9)

`transformers.trainer_utils.seed_worker` imported but unused

Remove unused import: `transformers.trainer_utils.seed_worker`

(F401)